### PR TITLE
feat: egov-icons

### PR DESCRIPTION
**Pull request by kronsolo**

### CODE REVIEW

File: src/Emoji/Angry.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from "react";
+
+export const Angry = () => (
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M12 2C10.0222 2 8.08879 2.58649 6.4443 3.6853C4.79981 4.78412 3.51809 6.3459 2.76121 8.17317C2.00433 10.0004 1.8063 12.0111 2.19215 13.9509C2.578 15.8907 3.53041 17.6725 4.92894 19.0711C6.32746 20.4696 8.10929 21.422 10.0491 21.8079C11.9889 22.1937 13.9996 21.9957 15.8268 21.2388C17.6541 20.4819 19.2159 19.2002 20.3147 17.5557C21.4135 15.9112 22 13.9778 22 12C22 10.6868 21.7413 9.38642 21.2388 8.17317C20.7363 6.95991 19.9997 5.85752 19.0711 4.92893C18.1425 4.00035 17.0401 3.26375 15.8268 2.7612C14.6136 2.25866 13.3132 2 12 2ZM6.9312 8.1055L10.1187 9.6992L9.2241 11.4883L6.0366 9.8945L6.9312 8.1055ZM14.8281 17.4555C14.0664 16.7278 13.0535 16.3217 12 16.3217C10.9465 16.3217 9.93363 16.7278 9.1719 17.4555L7.7568 16.041C8.31398 15.4837 8.97548 15.0416 9.70354 14.74C10.4316 14.4383 11.2119 14.2831 12 14.2831C12.7881 14.2831 13.5684 14.4383 14.2965 14.74C15.0245 15.0416 15.686 15.4837 16.2432 16.041L14.8281 17.4555ZM14.7759 11.4887L13.8813 9.6996L17.0688 8.1059L17.9634 9.8949L14.7759 11.4887Z" fill="black"/>
```
Comment (Zhassulan-Baigozha) [inline src/Emoji/Angry.tsx:5]: Format

File: src/General/Accessibility.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+export const Accessibility = () => (
+  <svg
+    width="24"
+    height="24"
+    viewBox="0 0 24 24"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path
+      fill-rule="evenodd"
```
Comment (Zhassulan-Baigozha) [inline src/General/Accessibility.tsx:12]: Unknown property 'fill-rule' found, use 'fillRule' insteadsonarlint (typescript:S6747)

Comment (Zhassulan-Baigozha) [reply src/General/Accessibility.tsx:12]: Unknown property 'clip-rule' found, use 'clipRule' insteadsonarlint (typescript:S6747)

File: src/General/Add.tsx [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import React from "react";
+
+export const Add = () => (
```
Comment (Zhassulan-Baigozha) [inline src/General/Add.tsx:3]: Format

File: src/Emoji/Angry.tsx (hunk 1 above)
Comment (kronsolo) [reply src/Emoji/Angry.tsx]: fixed

File: src/General/Add.tsx (hunk 3 above)
Comment (kronsolo) [reply src/General/Add.tsx:3]: fixed

**Pull request merged**